plugins/lookup/passwordstore.py

```diff
@@ -261,19 +261,21 @@ def check_pass(self):
                     if ':' in line:
                         name, value = line.split(':', 1)
                         self.passdict[name.strip()] = value.strip()
+            if os.path.isfile(os.path.join(self.paramvals['directory'], self.passname + ".gpg")):
+                # Only accept password as found, if there a .gpg file for it (might be a tree node otherwise)
+                password_found = True
```

Comment:
The logic will be a bit simpler if you simply `return True` here...
```suggestion
                return True
```

Reply:
If early returns are fine, I’m happy with this improvement

Reply:
I don't see why they should not :)